src/sos/actions.py

```diff
@@ -773,13 +768,14 @@ def sos_run(workflow=None,
                                 if not x.startswith("__"))
         if "ACTION" in env.config["SOS_DEBUG"] or "ALL" in env.config[
                 "SOS_DEBUG"]:
-            env.log_to_file(
-                "ACTION",
-                "Executing workflow ``{}`` with input ``{}`` and {}".format(
-                    workflow,
-                    short_repr(env.sos_dict.get("_input", None), True),
-                    "no args" if not args_output else args_output,
-                ),
+
+            into_log_to_file = (
```

Comment:
why this?

Reply:
I see what you are trying to do but `env.log_to_file` has two parematers....

Reply:
I resolved this very recently. env.log_to_file now takes two arguments: "ACTION" and into_log_to_file.